runtime/pom.xml

```diff
@@ -283,6 +283,13 @@
             <version>${codahale.metrics.version}</version>
         </dependency>
 
+        <!-- In-Memory Guava Cache dependencies-->
```

Comment:
This is not required.

Reply:
Removed maven dependency. It's already added as part of com.google.common.cache package
dependency.